www/install/createTables.sql

```diff
@@ -2402,7 +2402,7 @@ CREATE TABLE `password_expiration_excluded_users` (
 
 CREATE TABLE `security_token` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `token` varchar(255) NOT NULL,
+  `token` LONGTEXT NOT NULL,
```

Comment:
https://blogs.intuit.com/blog/2020/03/23/increased-lengths-for-oauth-2-0-fields/#:~:text=The%20new%20maximum%20lengths%20are,Access%20tokens%3A%204096%20characters
```suggestion
  `token` varchar(4096) NOT NULL,
```